source/core/src/main/com/csse3200/game/gamestate/GameState.java

```diff
@@ -43,4 +43,17 @@ public static boolean checkState() {
         logger.info("states: {} {} {} {}", quests, inventory, player, stats);
         return (quests == null || inventory == null || player == null || stats == null);
     }
+
+    /**
+     * Clears all the GameState contents and resets them to their original state.
+     */
+    public static void clearState() {
```

Comment:
This doesnt seem to be used

Reply:
is being used for something else but thanks for being so thorough :)